### PR TITLE
Removed break in bubble_sort.py

### PR DESCRIPTION
**Pull request by PRITI1999**
Removed the unnecessary usage of break and restored single entry single exit mechanism.

### CODE REVIEW

File: sorts/bubble_sort.py [hunk 1]
```diff
@@ -22,14 +22,15 @@ def bubble_sort(collection):
     True
     """
     length = len(collection)
-    for i in range(length-1):
-        swapped = False
+    sorted = False
```
Comment (cclauss) [inline sorts/bubble_sort.py]: There is really nothing wrong with using __break__.  It is part of the language.

__sorted__ is a builtin in Python so shadowing it with a variable name is a bad idea.

File: sorts/bubble_sort.py [hunk 2]
```diff
@@ -22,14 +22,15 @@ def bubble_sort(collection):
...
+    while sorted == False and i < n: # Stop iteration if the collection is sorted.
+        sorted = True 
         for j in range(length-1-i):
             if collection[j] > collection[j+1]:
-                swapped = True
+                sorted = False
                 collection[j], collection[j+1] = collection[j+1], collection[j]
-        if not swapped:
-            break  # Stop iteration if the collection is sorted.
+         i += 1
```
Comment (cclauss) [inline sorts/bubble_sort.py]: Is there an indentation error (even multiple of four) on this line?

File: sorts/bubble_sort.py [hunk 3]
```diff
@@ -22,14 +22,15 @@ def bubble_sort(collection):
     True
     """
     length = len(collection)
-    for i in range(length-1):
-        swapped = False
+    swapped = True
+    i = 0
+    while swapped == True and i < n: # Stop iteration if the collection is sorted.
```
Comment (poyea) [inline sorts/bubble_sort.py]: What is `n`? Do you mean `length`?

Comment (cclauss) [reply sorts/bubble_sort.py]: https://travis-ci.org/TheAlgorithms/Python/builds/586149305#L600

Comment (PRITI1999) [reply sorts/bubble_sort.py]: Thanks. Replaced n by length

**Comment by cclauss** [comment]
What is the advantage of implicating this PR?  It has more lines, more tests, and more complexity than the current code but I do not understand what problem that this change is solving.

**Pull request closed without merging**